server/package.json

```diff
@@ -53,7 +53,7 @@
     "archiver": "^5.3.1",
     "axios": "^0.26.0",
     "bcrypt": "^5.0.1",
-    "bullmq": "^3.14.1",
+    "bullmq": "^3.15.4",
```

Comment:
Should probably revert this as it's unrelated to the change in question.

Reply:
I was having build errors on buildmq 3.14.1
I didn't really debug as updating it fixed the issue.